src/components/Link.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Link as USWDSLink } from "@trussworks/react-uswds";
+import { Link as GatsbyLink } from "gatsby";
+
+// Pass only activeClassName and partiallyActive to GatsbyLink
+// Pass only fileType and size to file downloads
+const Link = ({
+  children,
+  to,
+  activeClassName,
+  partiallyActive,
+  variant,
+  fileType,
+  size,
+  ...other
+}) => {
+  // Assumes that any internal link will start with exactly one slash, and that anything else is external.
+  const internal = /^\/(?!\/)/.test(to);
+
+  // Assumed file download link structure
+  const file = /\.[0-9a-z]+$/i.test(to);
+
+  if (variant === "nav") {
+    // Navigation link
+    return (
+      // TODO: there's no special attribute/component that Gatsby has for navigation, so should Gatsby links be used here?
+      <USWDSLink href={to} variant="nav" {...other}>
+        {children}
+      </USWDSLink>
+    );
+  } else if (internal) {
```

Comment:
A file download is an internal link?

Reply:
This is how it is under 'Reminder: use <Link> only for internal links!' and 'File Downloads'. It kind of makes sense, too; file downloads will be for files in `public/` or `src/` folders.

https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/